spec/03-plugins/09-key-auth/02-access_spec.lua

```diff
@@ -180,7 +180,7 @@ for _, strategy in helpers.each_strategy() do
         })
         assert.res_status(401, res)
         local body = assert.res_status(401, res)
-        assert.equal([[{"message":"No API key found in request"}]], body)
+        assert.equal("{\n  \"message\":\"No API key found in request\"\n}", body)
```

Comment:
Let's not tie ourselves to those blank characters:
```suggestion
        assert.same({message = "No API key found in request"}, cjson.parse(body))
```